README.md

```diff
@@ -1,2 +1,32 @@
-# CDPLogger Client for Javascript
-A simple Javascript interface to communicate with CDP applications containing a CDPLogger component to retrieve historic data.
+# CDPLogger Client for JavaScript
+
+A simple JavaScript interface for communicating with CDP applications that include a CDPLogger component to retrieve historic data.
```

Comment:
Also add reference to cdpstudio.com and contact information like in the real-time data client https://github.com/CDPTechnologies/JavascriptCDPClient